### PR TITLE
updated_models

### PR DESCRIPTION
**Pull request by AlexK0210**

### CODE REVIEW

File: SummerProject/categories/models.py [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
-
-class SocialNetworkChoice(Enum):
-    FB = "Facebook"
-    TR = "Twitter"
-    GGL = "Google"
 
 class Category(models.Model):
     name = models.CharField(max_length=155, blank=False)
-    parent_category_id = models.ForeignKey('self', on_delete=models.CASCADE)
+    parent_category_id = models.ForeignKey("self", on_delete=models.CASCADE)
```
Comment (Liliia111) [inline SummerProject/categories/models.py:6]: you should add ```null=True``` to the ```parent_category_id``` field

File: SummerProject/static/src/router.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
 /* eslint max-len: ["error", { "ignoreStrings": true }]*/
 
 import React from 'react';
-import {Route, Switch, Redirect} from 'react-router-dom';
```
Comment (Liliia111) [inline SummerProject/static/src/router.js:4]: why did you remove ```Redirect``` from imports?

Comment (AlexK0210) [reply SummerProject/static/src/router.js:4]: I forgot to add it back :(
Gonna fix it ASAP
